src/Query.php

```diff
@@ -298,6 +300,44 @@ public function query($matcher)
     }
 
     /**
+     * Add a facet to the elastic query object
+     *
+     * @param  array|Elastica\Facet\AbstractFacet $facet One or multiple facets
+     * @return $this;
+     */
+    public function facet($facet)
```

Comment:
I would not include this at all. Facets is a deprecated concept in latests versions of elastic search